dotCMS/src/main/java/com/dotmarketing/business/APILocator.java

```diff
@@ -610,6 +611,10 @@ public static PublishingEndPointAPI getPublisherEndPointAPI() {
 	 * Creates a single instance of the {@link StructureAPI} class.
 	 *
 	 * @return The {@link StructureAPI} class.
+	 * @deprecated As of dotCMS 4.1.0, this API has been deprecated. From now on,
+	 *             please use the {@link ContentTypeAPI} class via
+	 *             {@link APILocator#getContentTypeAPI(User)} in order to interact
+	 *             with Content Types.
 	 */
 	public static StructureAPI getStructureAPI() {

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @Deprecated annotation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)